hazelcast-code-generator/src/main/java/com/hazelcast/client/protocol/generator/CodeGeneratorMessageTaskFactoryProvider.java

```diff
@@ -82,7 +82,7 @@ public void init(ProcessingEnvironment env) {
 
     @Override
     public boolean process(Set<? extends TypeElement> elements, RoundEnvironment env) {
```

Comment:
Now it is impossible for 3-rd party developer - integrated any client task factory.

To points:
        if (isEnterprise) {
            className = "EnterpriseMessageTaskFactoryProvider";
        } else if (isJet) {
            className = "JetMessageTaskFactoryProvider";
        } else {
            className = "DefaultMessageTaskFactoryProvider";
        }

2) public static final int JET_TEMPLATE_ID = 26;
Register of templateID is static

We need to discuss it with client team